src/django/Dockerfile

```diff
@@ -8,7 +8,10 @@ RUN set -ex \
     && buildDeps=" \
     build-essential \
     " \
-    && apt-get update && apt-get install -y $buildDeps --no-install-recommends \
+    && deps=" \
+    postgresql-client-12 \
```

Comment:
This is necessary because we install the PostgreSQL 11 client tooling in `django:2.2-python3.7-slim`. PostgreSQL 11 client tooling causes error for certain operations against a PostgreSQL 12 server.